typescript/infra/scripts/utils.ts

```diff
@@ -35,10 +35,22 @@ export function getArgsWithContext() {
     .alias('c', 'context');
 }
 
+export const modules = [
+  'core',
+  'igp',
+  'ica',
+  'iqs',
+  'create2',
+  'll',
+  'testrecipient',
+  'testquerysender',
+];
+export const sdkModules = ['core', 'igp', 'ica', 'iqs']; // TODO: check if this is correct
```

Comment:
I'd like to start writing TestRecipient artifacts as well but I think this is correct, feel free to remove the TODO